### DISCUSSION TITLE
vue-devui现在还不支持在Nuxt3中SSR吗？

### DISCUSSION THREAD

**Discussion by rd-nolan** [DevCloudFE/vue-devui#1530] [Q&A]
Nuxt3使用vue-devui做ssr时，发现vue-dev还不支持ssr？是我使用错了。还是？

**Comment by kagol** [comment]
@rd-carter 我尝试在 Nuxt3 项目中使用 Vue DevUI 组件是没问题的，你可以尝试以下方法：
在你的 Nuxt3 项目中创建一个 `plugins/vue-devui.ts` 文件，写入以下内容：
```
import VueDevUI from 'vue-devui';
import 'vue-devui/style.css';

export default defineNuxtPlugin((nuxtApp) => {
  nuxtApp.vueApp.use(VueDevUI);
});
```

然后在`app.vue`文件中就可以直接使用 Vue DevUI 的组件啦。
```
<template>
  <div>
    <DButton>确定</DButton>
    <DAlert type="success">操作成功！</DAlert>
    <NuxtWelcome />
  </div>
</template>
```

效果如下：

<img width="847" alt="image" src="https://user-images.githubusercontent.com/9566362/216079805-934cb0b5-a487-4198-86f4-f051df10429d.png">

在 Nuxt3 中使用 SSR，具体你是怎么使用的呢？遇到了什么问题呢？

**Reply by rd-nolan** [reply]
嗯嗯，使用的时候没问题。就是打包后，有点问题。我再研究研究哈
